### PR TITLE
Setup Appveyor

### PR DESCRIPTION
**Pull request by rahulporuri**
The appveyor related additions made in this PR are based on the appveyor setup in traitsui.

- the edm version used in 1.9.1 while travis uses 1.11.0
- tests are run on python 2.7, 3.5 and 3.6 with the toolkits null, pyqt on all runtimes and pyside, wx on python 2 specifically, like travis.

_Edit_: closes https://github.com/enthought/chaco/issues/377

### CODE REVIEW

**Comment by rahulporuri** [comment]
Note that both the 3.5 and 3.6 appveyor CI jobs finished running the tests successfully but they are marked as failures because of these tracebacks which appear when running the test suite

```
appveyor-run.cmd test %runtime% %toolkits%
Running tests in 'chaco-test-3.5-null'
copying file to tempdir: .coveragerc
[EXECUTING] edm run -e chaco-test-3.5-null -- coverage run -m unittest discover -v chaco
Could not covert font to FontEntry for file c:\windows\fonts\ebrima.ttf
Traceback (most recent call last):
  File "c:\users\appveyor\.edm\envs\chaco-test-3.5-null\lib\site-packages\kiva\fonttools\font_manager.py", line 1440, in <module>
    fontManager = pickle_load(_fmcache)
  File "c:\users\appveyor\.edm\envs\chaco-test-3.5-null\lib\site-packages\kiva\fonttools\font_manager.py", line 1050, in pickle_load
    fh = open(filename, 'rb')
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\appveyor\\AppData\\Roaming\\Enthought\\kiva\\fontList.cache'
```

**Comment by rahulporuri** [comment]
OTOH, on Python 2, we don't see the font-related tracebacks but we do see the following traceback after all of the testsuites are run, which I guess is being interpreted as a failed job

```
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "c:\users\appveyor\.edm\envs\chaco-test-2.7-wx\lib\atexit.py", line 24, in _run_exitfuncs
    func(*targs, **kargs)
PyAssertionError: C++ assertion "GetEventHandler() == this" failed at ..\..\src\common\wincmn.cpp(478) in wxWindowBase::~wxWindowBase(): any pushed event handlers must have been removed
Error in sys.exitfunc:
Traceback (most recent call last):
  File "c:\users\appveyor\.edm\envs\chaco-test-2.7-wx\lib\atexit.py", line 24, in _run_exitfuncs
    func(*targs, **kargs)
wx._core.PyAssertionError: C++ assertion "GetEventHandler() == this" failed at ..\..\src\common\wincmn.cpp(478) in wxWindowBase::~wxWindowBase(): any pushed event handlers must have been removed
```

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+build: false
+shallow_clone: false
+skip_branch_with_pr: true
+environment:
+
+  global:
+    PYTHONUNBUFFERED: "1"
+    INSTALL_EDM_VERSION: "1.9.1"
```
Comment (jvkersch) [inline appveyor.yml]: Is there any reason not to have the latest EDM here?

**Comment by jvkersch** [comment]
@rahulporuri Thanks for picking this up, this is great!

Do you have any spare cycles to investigate those two tracebacks further?

File: appveyor.yml (hunk 1 above)
Comment (rahulporuri) [reply appveyor.yml]: I remember there being issues with the existing `install-edm-windows.cmd` script and the latest edm version 1.11.0. I'll investigate this and get back to you.

**Comment by rahulporuri** [comment]
> Do you have any spare cycles to investigate those two tracebacks further?

I don't really know where the start.

**Comment by jvkersch** [comment]
> I don't really know where the start.

Hah, neither do I -- I was hoping you did :-) I've never seen the `FileNotFoundError`, but the C++ cleanup issue comes up on other platforms as well. I'll do some digging (time permitting) and maybe we can compare notes later on and see if we get anywhere?

**Comment by corranwebster** [comment]
The second traceback is a WxPython issue caused by not properly disconnecting event handlers.  If I recall, @RobinD42 fixed this in TraitsUI with this PR https://github.com/enthought/traitsui/pull/108 . A similar fix may be needed in Enable?

Both of these tracebacks are almost certainly Enable/Kiva problems not Chaco bugs.

File: appveyor.yml (hunk 1 above)
Comment (rahulporuri) [reply appveyor.yml]: yay! fixed the `install-edm-windows.cmd` script and we now use edm 1.11.0

**Comment by rahulporuri** [comment]
@jvkersch I haven't been able to spend time investigating the tracebacks in the CI. Are you okay with merging this PR and opening issues to investigate the tracebacks?

**Comment by jvkersch** [comment]
> Are you okay with merging this PR and opening issues to investigate the tracebacks?

Sounds good! Merge away.

**Comment by rahulporuri** [comment]
merging now that the webhook for appveyor has been added

**Pull request merged**